content/lessons/chapter-3/index.ts

```diff
@@ -0,0 +1,11 @@
+import * as Intro1 from './intro-1'
+import * as Intro2 from './intro-2'
+import * as YVB1 from './you-vs-bit-1'
+
+const Chapter3 = {
+  'intro-1': Intro1,
+  'intro-2': Intro2,
+  'you-vs-bit-1': YVB1,
```

Comment:
Lets rename `you-vs-bit-1` to `solo-1` and for the co-op and pool once we'll add later we can use `coop` and `pool`

Reply:
Sounds good! Let me rename it!

Reply:
Addressed in 5a1178b3952e7d024452fad6416a2a65c9fdba47
Thanks for the suggestion!